### PR TITLE
PREAPPS-6149:- Added blank locale properties file for cs(Czech), sk(Slovak).

### PR DESCRIPTION
**Pull request by madhavdhuppe**

### CODE REVIEW

**Comment by silentsakky** [comment]
Does this show these locales in the dropdown for language selection?

File: WebRoot/messages/ZtMsg_sk.properties [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#
```
Comment (silentsakky) [inline WebRoot/messages/ZtMsg_sk.properties:1]: it's ok to not add these files for ZtMsg, it's for sencha touch and it's deprecated/removed from zimbra 9

Comment (madhavdhuppe) [reply WebRoot/messages/ZtMsg_sk.properties:1]: sure

**Review by BhavinBathani — APPROVED** [review]
LGTM

**Comment by madhavdhuppe** [comment]
> Does this show these locales in the dropdown for language selection?

Yes

**Pull request merged**